src/common/snippets/include/snippets/lowered/expression.hpp

```diff
@@ -32,7 +32,7 @@ class Expression : public std::enable_shared_from_this<Expression> {
     std::shared_ptr<Emitter> get_emitter() const;
 
     RegInfo get_reg_info() const;
-    void set_reg_info(RegInfo rinfo);
+    void set_reg_info(const RegInfo& rinfo);
```

Comment:
`PortDescriptor` initializes the class field `m_reg` using `set_reg` call but `Expression` doesn't have the field `RegInfo` - this method just [propagates](https://github.com/a-sidorova/openvino/blob/26f1bcc749cdeb88fed8666ad67c4e385ea39f6e/src/common/snippets/src/lowered/expression.cpp#L85) values to `PortDescriptors`. I mean that probably it makes sense if you initialize the class field (`m_reg = std::move(reg);`). Otherwise we should work with constant ref. Correct me please If I missed something 😃

Reply:
Ok, never mind